drivers/fuel_gauge/max17048/max17048.c

```diff
@@ -169,6 +154,21 @@ static int max17048_init(const struct device *dev)
 		return -ENODEV;
 	}
 
+	/* 0mV (0x00) is the default value */
+	data->undervoltage_threshold =
+		DT_PROP_OR(DT_INST(0, DT_DRV_COMPAT), undervoltage_threshold, 0);
+	/* 5100mV (0xFF) is the default value */
+	data->overvoltage_threshold =
+		DT_PROP_OR(DT_INST(0, DT_DRV_COMPAT), overvoltage_threshold, 5100);
+	/* 4% (0x1C) is the default value */
+	data->low_soc_threshold = DT_PROP_OR(DT_INST(0, DT_DRV_COMPAT), low_soc_threshold, 4);
```

Comment:
If there are multiple instances, this will use the same value for all instances. You need to add `undervoltage_threshold`, `overvoltage_threshold`, and `low_soc_threshold` to `struct max17048_config` so you can get the correct value from devicetree for each instance.

If the default values here correspond to hardware register reset values, then you can set them in the binding instead. But make sure to document that in the property description.